lib/src/json_schema/json_schema.dart

```diff
@@ -184,55 +185,136 @@ class JsonSchema {
     return new JsonSchema._fromMap(_root, schemaDefinition, path);
   }
 
-  // Root Schema Properties
+  // --------------------------------------------------------------------------
+  // Root Schema Fields
+  // --------------------------------------------------------------------------
+
+  /// The root [JsonSchema] for this [JsonSchema].
   JsonSchema _root;
+
+  /// JSON of the [JsonSchema] as a [Map].
   Map<String, dynamic> _schemaMap = {};
+
+  /// A [List<JsonSchema>] which the value must conform to all of.
   List<JsonSchema> _allOf = [];
+
+  /// A [List<JsonSchema>] which the value must conform to at least one of.
   List<JsonSchema> _anyOf = [];
+
+  /// Default value of the [JsonSchema].
   dynamic _defaultValue;
+
+  /// Included [JsonSchema] definitions.
   Map<String, JsonSchema> _definitions = {};
+
+  /// Description of the [JsonSchema].
   String _description;
+
+  /// Possible values of the [JsonSchema].
   List _enumValues = [];
+
+  /// Whether the maximum of the [JsonSchema] is exclusive.
   bool _exclusiveMaximum;
+
+  /// Whether the minumum of the [JsonSchema] is exclusive.
   bool _exclusiveMinimum;
 
-  /// Support for optional formats (date-time, uri, email, ipv6, hostname)
+  /// Pre-defined format (i.e. date-time, email, etc) of the [JsonSchema] value.
   String _format;
+
+  /// ID of the [JsonSchema].
   Uri _id;
+
+  /// Maximum value of the [JsonSchema] value.
   num _maximum;
+
+  /// Minimum value of the [JsonSchema] value.
   num _minimum;
+
+  /// Maximum value of the [JsonSchema] value.
   int _maxLength;
+
+  /// Minimum length of the [JsonSchema] value.
   int _minLength;
+
+  /// The number which the value of the [JsonSchema] must be a multiple of.
   num _multipleOf;
+
+  /// A [JsonSchema] which the value must NOT be.
   JsonSchema _notSchema;
+
+  /// A [List<JsonSchema>] which the value must conform to at least one of.
   List<JsonSchema> _oneOf = [];
+
+  /// The regular expression the [JsonSchema] value must conform to.
   RegExp _pattern;
+
+  /// Ref to the URI of the [JsonSchema].
   String _ref;
+
+  /// The path of the [JsonSchema] within the root [JsonSchema].
   String _path;
+
+  /// Title of the [JsonSchema].
   String _title;
+
+  /// List of allowable types for the [JsonSchema].
   List<SchemaType> _schemaTypeList;
 
+  // --------------------------------------------------------------------------
```

Comment:
Is this how we want to do Code "Sections"? Been struggling to find dart consistency with these.

Reply:
I use the one's from Jace's over react snippets package. No opinion here, I'd be happy to settle on these.